### PR TITLE
Courtship song muscles and neurons

### LINKED ISSUE #240: New term request: hg1 & ps1 muscle and motor neurons
FBrf0223321 describes a sexually dimorphic wing muscle, hg1 and a monomorphic wing muscle, ps1, that are involved in courtship songs and the motor neurons that innervate them.

### PR DESCRIPTION
**Pull request by Clare72**
Addition of hg1 and hg4 muscles and hg1 and ps1 motor neurons.
More synonyms for other wing muscles.
Some reorganisation of wing muscle and associated motor neuron classes.
fixes #240

### CODE REVIEW

File: fbbt/src/ontology/fbbt-edit.obo [hunk 1]
```diff
@@ -30755,50 +30750,56 @@ is_a: FBbt:00003360 ! flight muscle
 
 [Term]
 id: FBbt:00003380
-name: direct flight muscle 49
-def: "Tergo-pleural muscle extending from the anterior notum to the basalar of the wing (Sink, 2006). It serves to draw the wing horizontally forwards (Miller, 1950). It is innervated by the b1 motor neuron." [FlyBase:FBrf0007735, FlyBase:FBrf0064793, FlyBase:FBrf0074636, ISBN:978-0-387-30053-5]
+name: basalar muscle 49
+def: "Muscle extending from the anterior notum to the basalar of the wing (Sink, 2006). It serves to draw the wing horizontally forwards (Miller, 1950). It is innervated by the b1 motor neuron." [FlyBase:FBrf0007735, FlyBase:FBrf0064793, FlyBase:FBrf0074636, ISBN:978-0-387-30053-5]
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Maybe add comment noting previous, incorrect name & classification.

File: fbbt/src/ontology/fbbt-edit.obo [hunk 2]
```diff
@@ -68686,6 +68681,91 @@ relationship: overlaps FBbt:00007053 ! lateral horn
...
+[Term]
+id: FBbt:00047244
+name: external muscle of fourth axillary hg1
+def: "Direct flight muscle that inserts into the fourth axillary (posterior notal wing process) and functions in carrying out lateral tip movements (Wisser and Nachtigall, 1984). It shares a tendon with the external muscle of fourth axillary hg4 (Williams and Williams, 1943). In contrast to most other wing muscles, hg1 is larger in males than in females, which is thought to allow for louder sine song (Shirangi et al., 2013). It is one of only two flight muscles that express doublesex (Shirangi et al., 2013)." [doi:10.1007/BF00312139, FlyBase:FBrf0006004, FlyBase:FBrf0098527, FlyBase:FBrf0223321]
+comment: This muscle is not illustrated in Miller 1950 (FBrf0007735).
+synonym: "hg1 muscle" EXACT []
+synonym: "posterior notal wing process muscle hg1" EXACT [FlyBase:FBrf0098527]
+is_a: FBbt:00003387 ! axillary muscle
+relationship: expresses http://flybase.org/reports/FBgn0000504
+relationship: synapsed_by FBbt:00047246 ! hg1 motor neuron
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Please add 
subClassOf: 'capable of' some 'male courtship behavior, veined wing generated song production' (GO:0045433)
Ditto hg4 & hg1.

File: fbbt/src/ontology/fbbt-edit.obo [hunk 3]
```diff
@@ -68686,6 +68681,91 @@ relationship: overlaps FBbt:00007053 ! lateral horn
...
+intersection_of: synapsed_to FBbt:00003360 ! flight muscle
+relationship: has_soma_location FBbt:00111084 ! cell body rind of mesothoracic neuromere
+created_by: CP
+creation_date: 2018-06-14T13:59:26Z
+
+[Term]
+id: FBbt:00047248
+name: ps1 motor neuron
+def: "Motor neuron that innervates the mesothoracic pleurosternal muscle 59 (pleurosternal 1). This cell plays a role in pulse carrier frequency and pulse amplitude (Shirangi et al., 2013)." [FlyBase:FBrf0223321]
+is_a: FBbt:00005123 ! motor neuron
```
Comment (dosumis) [inline fbbt/src/ontology/fbbt-edit.obo]: Please add
subClassOf: 'capable of' some 'male courtship behavior, veined wing generated song production' (GO:0045433)

**Comment by dosumis** [comment]
Great to see this cleaned up & extended.  Couple of minor issues to address (see comments in-line)  Then happy for you to merge.

File: fbbt/src/ontology/fbbt-edit.obo (hunk 2 above)
Comment (Clare72) [reply fbbt/src/ontology/fbbt-edit.obo]: not capable_of_part_of? (if neurons and muscles are each doing part)

**Comment by dosumis** [comment]
All good.  Please merge.

**Pull request merged**